utils/_context/_scenarios/__init__.py

```diff
@@ -335,6 +335,18 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.APPSEC],
     )
 
+    iast_standalone = EndToEndScenario(
```

Comment:
Could you add this new scenario  in `.github/workflows/run-end-to-end.yml`?

Reply:
Do you mean add it to `scenario_groups.yml` and `run-end-to-end.yml` as it was done for appsec_standalone?
I've done it in https://github.com/DataDog/system-tests/pull/3313/commits/3363cb5d53b6dea02f7f371d6282de56eaab7580